tests/Func/ViewTeamTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Func;
+
+use App\Team\Domain\Repository\MemberRepository;
+use Symfony\Component\DomCrawler\Crawler;
+
+final class ViewTeamTest extends FunctionalTestCase
+{
+    public function testRenderProperHtml(): void
+    {
+        $this->get('/team');
+
+        $this->assertSelectorTextContains('h1', 'team.title');
+        $this->assertSelectorExists('[data-test-review]');
+    }
+
+    public function testEveryTeamMemberHasHisModal(): void
+    {
+        $this->get('/team');
+
+        foreach ($this->getService(MemberRepository::class)->findAll() as $member) {
+            $this->assertSelectorExists(sprintf('[data-member-modal-member-value="%s"]', $member->id->value));
+        }
+    }
+
+    public function testEveryTeamMemberIsListedAsReviewer(): void
+    {
+        $this->get('/team');
+
+        foreach (array_values($this->getService(MemberRepository::class)->findAll()) as $i => $member) {
+            $this->assertSelectorTextContains(sprintf('[data-test-reviewers] ul > li:nth-child(%d) > p', $i + 1), $member->getFullname());
```

Comment:
Let's use a dedicated CSS selectors instead for the reason described in [Selecting Elements best practices](https://docs.cypress.io/app/core-concepts/best-practices#Selecting-Elements)
```suggestion
            $this->assertSelectorTextContains(sprintf('[data-test-reviewers] [data-test-member="%d"] [data-test-member-full-name]', $i + 1), $member->getFullname());
```